contracts/gateway/src/events.rs

```diff
@@ -1,24 +1,26 @@
-use connection_router::events::make_message_event;
-use connection_router::state::Message;
+use connection_router::events::make_message_event_new;
+use connection_router::state::NewMessage;
 use cosmwasm_std::Event;
 
 pub enum GatewayEvent {
-    MessageVerified { msg: Message },
-    MessageVerificationFailed { msg: Message },
-    MessageRouted { msg: Message },
-    MessageRoutingFailed { msg: Message },
+    MessageVerified { msg: NewMessage },
+    MessageVerificationFailed { msg: NewMessage },
+    MessageRouted { msg: NewMessage },
+    MessageRoutingFailed { msg: NewMessage },
 }
 
 impl From<GatewayEvent> for Event {
     fn from(other: GatewayEvent) -> Self {
         match other {
-            GatewayEvent::MessageVerified { msg } => make_message_event("message_verified", msg),
-            GatewayEvent::MessageRouted { msg } => make_message_event("message_routed", msg),
+            GatewayEvent::MessageVerified { msg } => {
+                make_message_event_new("message_verified", msg)
```

Comment:
this function name is so odd.

Reply:
the `new` is tagged on to differentiate the existing function which I can't delete yet. Once the old message types are deleted, this will become `make_message_event`